adhocracy4/polls/assets/Question.jsx

```diff
@@ -1,13 +1,10 @@
-var React = require('react')
-var django = require('django')
-var Alert = require('../../../static/Alert')
-var update = require('immutability-helper')
+const React = require('react')
```

Comment:
In my current task I am restructuring this part. Actually I just prepend a Questions (list) component to this.
The action List Item (Question.jsx) i didn't touch yet, so this one here is way more refactored and advanced.

Long story short, how can we get this together?

Reply:
@phillimorland

Reply:
I think we will merge this and then port your (@khamui) changes over and after that do all the changes in A4 right away!

Reply:
it should be be fine to just whack it in